opentelemetry_collector/receiver/metricgenerator/metric_generator.go

```diff
@@ -29,6 +30,13 @@ func MakeInt64TimeSeries(val int64, startTime, now time.Time, labels []*metricsp
 	}
 }
 
+// MakeExponentialBucketOptions generates a proto representation of a config which,
+// defines a distribution's bounds. This defines maxExponent + 1 (= N) buckets. The boundaries for bucket
```

Comment:
Not sure whether `maxExponent + 1 (= N)` is the right expression. It confuses me at first. Do we need to introduce `N`? How about 

```
This defines maxExponent + 1 buckets. ....
...
[boundsBase ^ (i - 1)], boundsBase ^ i) for 0 < i < maxExponent
[boundsBase ^ i, +infinity) for i == maxExponent
```

Reply:
I've removed the reference to N and also fixed an error in comment's explanation.